ios/01-query-based-sync/ToDo/ProjectsViewController.swift

```diff
@@ -31,7 +31,8 @@ class ProjectsViewController: UIViewController, UITableViewDelegate, UITableView
     let activityIndicator = UIActivityIndicatorView()
     
     override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
-        realm = try! Realm(configuration: SyncConfiguration.automatic())
+        let config = SyncUser.current?.configuration()
+        realm = try! Realm(configuration: config!)
```

Comment:
`realm = try! Realm(configuration: SyncUser.current!.configuration())`

Optional chaining off `SyncUser.current` rather than forcing it there just makes it less obvious where things failed if `SyncUser.current` does return nil.